openapi-generator/build.gradle.kts

```diff
@@ -69,7 +67,7 @@ arrayOf(
 	"stable-pre" to "Prerelease",
 	"unstable" to "Unstable",
 ).forEach { (flavor, flavorPascalCase) ->
-	tasks.register("downloadApiSpec${flavorPascalCase}", Download::class) {
+	tasks.register("downloadApiSpec${flavorPascalCase}", de.undercouch.gradle.tasks.download.Download::class) {
```

Comment:
You might still want to keep the import here, I think.

Reply:
Nah, no imports in build files

Reply:
Hmm ok. I personally prefer them over those super long identifiers but ok.